feature/home/src/main/java/com/epmedu/animeal/home/presentation/ui/MapboxMap.kt

```diff
@@ -1,34 +1,40 @@
 package com.epmedu.animeal.home.presentation.ui
 
 import androidx.compose.foundation.layout.fillMaxSize
-import androidx.compose.runtime.Composable
-import androidx.compose.runtime.LaunchedEffect
-import androidx.compose.runtime.getValue
-import androidx.compose.runtime.mutableStateOf
-import androidx.compose.runtime.remember
+import androidx.compose.runtime.*
 import androidx.compose.runtime.saveable.rememberSaveable
-import androidx.compose.runtime.setValue
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.viewinterop.AndroidView
+import com.epmedu.animeal.home.presentation.OnFeedingPointClickListener
+import com.epmedu.animeal.home.presentation.model.FeedingPointUi
 import com.epmedu.animeal.home.presentation.viewmodel.HomeState
+import com.epmedu.animeal.home.utils.MarkerCache
 import com.mapbox.geojson.Point
 import com.mapbox.maps.CameraOptions
 import com.mapbox.maps.MapInitOptions
 import com.mapbox.maps.MapView
 import com.mapbox.maps.ResourceOptions
+import com.mapbox.maps.plugin.annotation.annotations
+import com.mapbox.maps.plugin.annotation.generated.*
 import com.mapbox.maps.plugin.locationcomponent.location
 import com.mapbox.maps.plugin.scalebar.scalebar
 
 @Composable
-fun MapboxMap(state: HomeState) {
+fun MapboxMap(
+    state: HomeState,
+    onFeedingPointClickListener: OnFeedingPointClickListener
+) {
     val mapBoxView = mapView(state.mapBoxPublicKey, state.mapBoxStyleUri)
 
     var initialLocationReceived by rememberSaveable { mutableStateOf(false) }
     LaunchedEffect(Unit) { initialLocationReceived = false }
 
     AndroidView(
-        factory = { mapBoxView },
+        factory = {
+            addMarkers(mapBoxView, state.feedingPoints, onFeedingPointClickListener)
```

Comment:
please check following case: add `delay` inside flow for some seconds and check if markers appeared in screen

Reply:
If I add the delay it does not show the markers… How about moving it to the update section instead of factory but restrict it with some flag in the HomeState? Like:

```
if(state.arePointsLoaded){
	addMarkers()
}
```

Reply:
Factory method can help

`state.arePointsDisplayed` looks like workaround. Need to check if markers automatically removed after changing data

I suggest to test in small portion of data (like 4 markers):

Init `flow` with following structure:
```
flowOf {
    4 markers
    delay(5sec)
    new 4 markers
}
```
And them check on the screen result (accepted behaviors is only last 4 markers on the map)

In other cases we need to handle it (like remove markers before each update or something like this)